docs/VLCB4in4out_README.md

```diff
@@ -0,0 +1,183 @@
+# VLCB4IN4OUT
+
+An Arduino program to allocate 4 switches as input pins and 4 outputs to LEDs.
+
+Key Features:
+- MERG VLCB interface.
+- LED flash rate selectable from event variables.
+- Switch function controllable by node variables.
+- Modular construction to ease adaptation to your application.
+
+## Overview
+
+The program is written in C++ but you do not need to understand this to use the program.
+The program includes a library that manages the LED functionality.
+
+## Using VLCB4in4out
+
+The MCP2515 interface requires five Arduino pins to be allocated. Three of these are fixed
+in the architecture of the Arduino processor. One pin must be connected to an interrupt
+capable Arduino pin. The Chip Select pin can be freely defined. This example is configured
+for use with an Uno but can easily be adapted for use with other Arduino types.
```

Comment:
I think it would be appropriate to describe what the fixed pins are and what pins are used in the sketch for the ones you can change. Also for pins like INT you can only choose pin 2 or 3.
You may also make a reference to to your shield where certain pins are used.